src/muya/lib/contentState/paragraphCtrl.js

```diff
@@ -192,8 +192,7 @@ const paragraphCtrl = ContentState => {
     const startParents = this.getParents(startBlock)
     const endParents = this.getParents(endBlock)
     const hasFencedCodeBlockParent = () => {
-      return startParents.some(b => b.type === 'pre' && /code/.test(b.functionType)) ||
-        endParents.some(b => b.type === 'pre' && /code/.test(b.functionType))
+      return [...startParents, ...endParents].some(b => b.type === 'pre' && /code/.test(b.functionType))
```

Comment:
I think the previous solution was more effective because no copy is made. The new code will create a new array (copies both arrays) and checks afterwards.

Reply:
In fact, in javascript, the performance impact is very small, the latter method only use one `some` functions,  which performance maybe better, I have not tested, I just think that the latter way, the code is less.

Reply:
The new solution uses 2 for loops to copy elements and creates a temporary array. The old solution does not but merge if you want.